components/apiserver-proxy/internal/proxy/proxy.go

```diff
@@ -9,13 +9,18 @@ import (
 	"github.com/golang/glog"
 	"github.com/kyma-project/kyma/components/apiserver-proxy/internal/authn"
 	"github.com/kyma-project/kyma/components/apiserver-proxy/internal/authz"
+
 	yaml "gopkg.in/yaml.v2"
+	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/apiserver/pkg/authentication/authenticator"
 	"k8s.io/apiserver/pkg/authentication/user"
 	"k8s.io/apiserver/pkg/authorization/authorizer"
+	apirequest "k8s.io/apiserver/pkg/endpoints/request"
 	clientset "k8s.io/client-go/kubernetes"
 )
 
+const KUBERNETES_SERVICE = "kubernetes.default"
```

Comment:
This constant seems to be unused. If so, please remove it.

Reply:
It is used in main.go